The Grand ReReReReReopening the final straw part 9: Intra-Particle-Force Surface Tension model (#434)
trixi-framework/TrixiParticles.jl: docs/src/systems/weakly_compressible_sph.md

The work by Akinci proposes three forces:
- a cohesion force
- a surface area minimization force
- a wall adhesion force

The classical model is composed of the curvature minimization and cohesion force.

#### Cohesion force
The model calculates the cohesion force based on the distance between particles and the smoothing length.
This force is determined using two distinct regimes within the support radius:
- For particles closer than half the support radius,
  a repulsive force is calculated to prevent particle clustering too tightly,
  enhancing the simulation's stability and realism.
- Beyond half the support radius and within the full support radius,
  an attractive force is computed, simulating the effects of surface tension that draw particles together.
The cohesion force, ``F_{\text{cohesion}}``, for a pair of particles is given by:
```math
F_{\text{cohesion}} = -\sigma m_b C \frac{r}{\Vert r \Vert},
```
where:
- ``\sigma`` represents the surface tension coefficient, adjusting the overall strength of the cohesion effect.
- ``C`` is a scalar function of the distance between particles.

The Cohesion Kernel ``C`` is defined as 
```math

> \mathbf{C(r)}=\frac{32}{\pi h^9}

Reviewer: Still capitalized and bold.

\begin{cases}
(h-r)^3 r^3, & \text{if } 2r > h \\
2(h-r)^3 r^3 - \frac{h^6}{64}, & \text{if } r > 0 \text{ and } 2r \leq h \\
0, & \text{otherwise}
\end{cases}
```

#### Surface area minimization force
To model the minimization of the surface area and curvature of the fluid a curvature force is used which is calculated
```math
F_{\text{curvature}} = -\sigma (n_a - n_b)
```

#### Wall Adhesion Force
The wall adhesion model proposed by Akinci et al. is based on a kernel function which is 0 from 0.0 to 0.5 support radiia with a maximum at 0.75.
With the force calculated with an adhesion coefficient ``\beta`` as 
```math
F_{\text{adhesion}} = -\beta m_b A \frac{r}{\Vert r \Vert},
```
with ``A`` being the adhesion kernel defined as
```math

> \mathbf{A}(r)= \frac{0.007}{h^{3.25}}

Reviewer: Still bold in the second equation.

## [Surface Tension](@id surface_tension)

> ### Akinci based intra-particle force surface tension and wall adhesion model
> The work by Akinci proposes three forces:
> - a cohesion force
> - a surface normal force

Reviewer: ```suggestion
### Akinci-based intra-particle force surface tension and wall adhesion model
The work by Akinci proposes three forces:
- a cohesion force
- a surface normal force
```
And shouldn't "surface normal force" be "curvature minimization force"? I find it confusing that you list three forces here and explain a different force below.

- a wall adhesion force

The classical model is composed of the curvature minimization and cohesion force.

#### Cohesion Force
The model calculates the cohesion force based on the distance between particles and the smoothing length.
This force is determined using two distinct regimes within the support radius:
- For particles closer than half the support radius,
  a repulsive force is calculated to prevent particle clustering too tightly,
  enhancing the simulation's stability and realism.
- Beyond half the support radius and within the full support radius,
  an attractive force is computed, simulating the effects of surface tension that draw particles together.
The cohesion force, ``F_{\text{cohesion}}``, for a pair of particles is given by:
```math
F_{\text{cohesion}} = -\sigma m_b C \frac{r}{\Vert r \Vert},
```
where:
- ``\sigma`` represents the surface tension coefficient, adjusting the overall strength of the cohesion effect.
- ``C`` is a scalar function of the distance between particles.

#### Curvature Minimization Force
To model the minimization of the surface curvature of the fluid a curvature force is used which is calculated
```math

> F_{\text{surf_min}} = -\sigma (n_a - n_b)

Reviewer: ```suggestion
F_{\text{surf\_min}} = -\sigma (n_a - n_b)
```
Doesn't render.

```

#### Wall Adhesion Force
The wall adhesion model proposed by Akinci et al. is based on a kernel function which is 0 from 0.0 to 0.5 support radiia with a maximum at 0.75.

> With the force calculated with a adhesion coefficient ``\beta``: 

Reviewer: ```suggestion
With the force calculated with an adhesion coefficient ``\beta`` as 
```

- ``\sigma`` represents the surface tension coefficient, adjusting the overall strength of the cohesion effect.
- ``C`` is a scalar function of the distance between particles.

The Cohesion Kernel ``C`` is defined as 
```math
\mathbf{C(r)}=\frac{32}{\pi h^9}
\begin{cases}
(h-r)^3 r^3, & \text{if } 2r > h \\
2(h-r)^3 r^3 - \frac{h^6}{64}, & \text{if } r > 0 \text{ and } 2r \leq h \\
0, & \text{otherwise}
\end{cases}
```

#### Surface area minimization force
To model the minimization of the surface area and curvature of the fluid a curvature force is used which is calculated
```math
F_{\text{curvature}} = -\sigma (n_a - n_b)
```

> #### Wall Adhesion Force

Reviewer: Now it's `Wall adhesion Force`.
